tests/unit/amo/api/test_collections.js

```diff
@@ -287,4 +288,95 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('updateCollection', () => {
+    const _params = (params = {}) => {
```

Comment:
We usually call these `defaultParams()` and I think that's a bit more instructive.